libraries/standard/mqtt/include/mqtt.h

```diff
@@ -406,6 +406,26 @@ MQTTStatus_t MQTT_ReceiveLoop( MQTTContext_t * pContext,
  */
 uint16_t MQTT_GetPacketId( MQTTContext_t * pContext );
 
+/**
+ * @brief Parses the payload of an MQTT SUBACK packet that contains the status
+ * code of subscription request made for topic filters in the equivalent
+ * subscribe packet.
+ *
+ * This function can be used to iterate over the status code
+ *
+ * @param[in] pSubackPacket The SUBACK packet whose payload is to be parsed.
+ * @param[out] pPayloadStart This is populated with the starting address
+ * of the payload (or return codes for topic filters) in the SUBACK packet.
+ * @param[out] pPayloadSize This is populated with the size of the payload
+ * in the SUBACK packet. It represents the number of topic filters whose
+ * SUBACK status is present in the packet.
+ *
```

Comment:
Maybe we should add a note here about what values each status code can take, and what meaning they have.

Reply:
Added